Ref.Windows/MainWindow.xaml

```diff
@@ -274,32 +278,48 @@
+                    <ComboBoxItem>Licentiate Thesis</ComboBoxItem>
+                    <ComboBoxItem>Master's Thesis</ComboBoxItem>
+                </ComboBox>
+                <!-- Thesis School -->
+                <TextBlock Text="School" Visibility="{Binding School, Converter={StaticResource MakeVisible}, FallbackValue=Collapsed}"
+                           Grid.Row="15" Grid.Column="0" VerticalAlignment="Center" Margin="0,0,0,4" />
+                <TextBox Visibility="{Binding School, Converter={StaticResource MakeVisible}, FallbackValue=Collapsed}"
+                         Text="{Binding School}" IsReadOnly="{Binding IsReadOnly}"
+                         Grid.Row="15" Grid.Column="2" FontSize="14" Margin="4,0,0,4" />
                 <!-- ISBN -->
                 <TextBlock Text="ISBN" Visibility="{Binding Isbn, Converter={StaticResource MakeVisible}, FallbackValue=Collapsed}"
-                           Grid.Row="14" Grid.Column="0" VerticalAlignment="Center" Margin="0,0,0,4" />
+                           Grid.Row="16" Grid.Column="0" VerticalAlignment="Center" Margin="0,0,0,4" />
                 <TextBox Visibility="{Binding Isbn, Converter={StaticResource MakeVisible}, FallbackValue=Collapsed}"
                          Text="{Binding Isbn}" IsReadOnly="{Binding IsReadOnly}"
-                         Grid.Row="14" Grid.Column="2" FontSize="14" Margin="4,0,0,4" />
+                         Grid.Row="16" Grid.Column="2" FontSize="14" Margin="4,0,0,4" />
                 <!-- ISSN -->
                 <TextBlock Text="ISSN" Visibility="{Binding Issn, Converter={StaticResource MakeVisible}, FallbackValue=Collapsed}"
-                           Grid.Row="15" Grid.Column="0" VerticalAlignment="Center" Margin="0,0,0,4" />
+                           Grid.Row="17" Grid.Column="0" VerticalAlignment="Center" Margin="0,0,0,4" />
                 <TextBox Visibility="{Binding Issn, Converter={StaticResource MakeVisible}, FallbackValue=Collapsed}"
                          Text="{Binding Issn}" IsReadOnly="{Binding IsReadOnly}"
-                         Grid.Row="15" Grid.Column="2" FontSize="14" Margin="4,0,0,4" />
+                         Grid.Row="17" Grid.Column="2" FontSize="14" Margin="4,0,0,4" />
                 <!-- DOI -->
                 <TextBlock Text="DOI" Visibility="{Binding Doi, Converter={StaticResource MakeVisible}, FallbackValue=Collapsed}"
-                           Grid.Row="16" Grid.Column="0" VerticalAlignment="Center" Margin="0,0,0,4" />
+                           Grid.Row="18" Grid.Column="0" VerticalAlignment="Center" Margin="0,0,0,4" />
                 <TextBox Visibility="{Binding Doi, Converter={StaticResource MakeVisible}, FallbackValue=Collapsed}"
                          Text="{Binding Doi}" IsReadOnly="{Binding IsReadOnly}"
-                         Grid.Row="16" Grid.Column="2" FontSize="14" Margin="4,0,0,4" />
+                         Grid.Row="18" Grid.Column="2" FontSize="14" Margin="4,0,0,4" />
```

Comment:
To do: Get this grid mess sorted out. I've been too lazy to find out how to create a custom control (or like) for this. Manually keeping the rows in sync is starting to be painful.